### PR TITLE
Specify that Teams from the same organization must be grouped by Registration

### PR DESCRIPTION
**Pull request by jonahgreenthal**

### CODE REVIEW

File: registration.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: Registration
 ---
 A `Registration` object describes one school or other organization's entry into a tournament. It comprises the basic details about an organization and one or more [`Team`][team] objects.
 
+It is expected that each `Registration` object represents a distinct school or other organization. For instance, if "Anywhere High School" enters two teams, there should not be two `Registration` objects `name`d `Anywhere A` and `Anywhere B` and one `Team` child each, there should be one `Registration` object `name`d `Anywhere` (or `Anywhere High School`) and two `Team` children.
```
Comment (puls) [inline registration.md]: Can we do "with `name` fields" instead of "`name`d"?

File: css/main.css [hunk 2]
```diff
@@ -448,5 +448,8 @@ table.fields tr.required.annotation td:before {
   content: "*"
 }
 table.fields tr.required th:after {
-  content: "*"
+  content: " [req]"
+}
+table.fields tr.optional th:after {
+  content: " [opt]"
```
Comment (puls) [inline css/main.css]: nice.

File: registration.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ title: Registration
 ---
 A `Registration` object describes one school or other organization's entry into a tournament. It comprises the basic details about an organization and one or more [`Team`][team] objects.
 
+It is expected that each `Registration` object represents a distinct school or other organization. For instance, if "Anywhere High School" enters two teams, there should not be two `Registration` objects whose `name` fields are `Anywhere A` and `Anywhere B` and one `Team` child each, there should be one `Registration` object whose `name` field is `Anywhere` (or `Anywhere High School`) and two `Team` children.
```
Comment (puls) [inline registration.md]: I think you want a semicolon after the second "each". Or maybe a "but".

Comment (jonahgreenthal) [inline registration.md]: I think this is correct as written (compare to "Don't go to Tarshish, go to Nineveh") but I'm not completely sure (I don't know grammatical rules, I just go by what sounds right to me, and am nearly always correct) and I'm fine with semicoloning it. Coming right up…

File: css/main.css (hunk 2 above)
Comment (jonahgreenthal) [inline css/main.css]: I actually didn't mean this to be part of this pull request, but I'm glad you approve.

**Pull request merged**